osu.Server.Queues.ScoreStatisticsProcessor/Models/SoloScore.cs

```diff
@@ -17,43 +18,72 @@ public class SoloScore
         [ExplicitKey]
         public ulong id { get; set; }
 
-        public int user_id { get; set; }
+        public uint user_id { get; set; }
 
-        public int beatmap_id { get; set; }
+        public uint beatmap_id { get; set; }
 
-        public int ruleset_id { get; set; }
+        public ushort ruleset_id { get; set; }
+
+        public bool has_replay { get; set; }
+        public bool preserve { get; set; }
+        public bool ranked { get; set; } = true;
```

Comment:
What is this going to be? Is this future-proofing for potential future "unranking" of scores?

Reply:
yeah